src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -547,12 +547,13 @@ public static string ConvertOldStringsToClass(string set, string category)
                 strLength -= 2;
             }
 
-#if REGEXGENERATOR
-            return StringExtensions.Create
+            return
+#if NETCOREAPP2_1_OR_GREATER
```

Comment:
As we don't compile netcoreapp2.1 anymore, would it make sense to use `#if NETCOREAPP` instead?

Reply:
I prefer to use the version in which they're available.